Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -423,6 +425,10 @@ public Item GetFocusTarget()
             if (positionOut == null) { return null; }
 
             item.SendSignal(new Signal(MathHelper.ToDegrees(targetRotation).ToString("G", CultureInfo.InvariantCulture), sender: user), positionOut);
+            if (item.Connections?.Find(c => c.Name == "distance_out") is { } distanceOut)
+            {
+                item.SendSignal(new Signal((targetDistance / 100).ToString("G", CultureInfo.InvariantCulture), sender: user), distanceOut);
```

Comment:
It's not obvious why the distance is divided by 100 here. Using `ConvertUnits.ToSimUnits` would make it more clear..

Reply:
This also creates new string objects every frame - it would be more efficient to just store the distance as a string when caching it.